setup.py

```diff
@@ -51,6 +51,7 @@ def get_version():
     extras_require={
         "airflow": ["apache-airflow==2.2.4"],
         "data": ["dominodatalab-data>=0.1.0"],
+        "flyte": ["flytekit>=1.10.2"],
```

Comment:
```suggestion
        "flyte": ["flytekit>=1.10.7"],
```
See https://dominodatalab.atlassian.net/browse/DOM-54691